packages/web-components/fast-foundation/src/horizontal-scroll/horizontal-scroll.ts

```diff
@@ -312,6 +312,23 @@ export class FASTHorizontalScroll extends FASTElement {
         this.setFlippers();
     }
 
+    /**
+     * Checks to see if the stops are returning values
+     *  otherwise it will try to reinitialize them
+     *
+     * @returns boolean for the current state of scrollStops are non-zero values
```

Comment:
This sentence is confusing and could use rephrasing.  Maybe something like "boolean indicating that current scrollStops are valid non-zero values"?

Reply:
Updated comments.